elasticsearch/prep-install.prod

```diff
@@ -1,11 +0,0 @@
-# prep the plugins we need
```

Comment:
If we no longer are using this script don't we need to make some adjustment either in the origin version or have this version for production?

Reply:
My plan is to have a prep-install.prod that lives only in the downstream repo